bert_reranker/main.py

```diff
@@ -34,14 +39,17 @@ def main():
     parser.add_argument('--output', help='where to store models', required=True)
     parser.add_argument('--no-model-restoring', help='will not restore any previous model weights ('
                                                      'even if present)', action='store_true')
-    parser.add_argument('--validation-only', help='will not train - will just evaluate on dev',
+    parser.add_argument('--train', help='will not train - will just evaluate on dev',
+                        action='store_true')
+    parser.add_argument('--validation', help='will not train - will just evaluate on dev',
                         action='store_true')
+    parser.add_argument('--predict', help='will predict on the json file you provide as an arg')
     args = parser.parse_args()
 
     logging.basicConfig(level=logging.INFO)
 
-    sys.stdout = LoggerWriter(logger.info)
-    sys.stderr = LoggerWriter(logger.warning)
+    #  sys.stdout = LoggerWriter(logger.info)
```

Comment:
weird - in the master branch now there is an if here - so that his part can be enabled with --redirect-log.
Could you check why this is not imported?